metadb/rqlite/disco/client.go

```diff
@@ -7,9 +7,10 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
-	"os"
+
```

Comment:
```suggestion
	"github.com/pastelnetwork/gonode/common/errors"
```
Again, Since we have to maintain this code and catch bugs and errors, it is preferable to have a stack error of the possible errors.

Reply:
done